packages/vite-plugin-checker/package.json

```diff
@@ -84,8 +84,10 @@
     "@types/lodash.pick": "^4.4.6",
     "@volar/vue-typescript": "^0.33.0",
     "esbuild": "^0.14.27",
+    "meow": "^9.0.0",
```

Comment:
Sorry for the late response. I had quite a busy bi-month and just started to get back to reviewing this big and important PR.  `meow` and `stylelint` should also be added to `peerDependencies` and marked as `optional`. Without this, in a pnpm monorepo project which set `hoist=false`, vite-plugin-checker will no longer able to access them.

Reply:
Added.